### ISSUE TITLE
How to classify mails as spam or ham?

### ISSUE THREAD

**Issue by phdoerfler** [r-raymond/nixos-mailserver#91]
We neither use the dovecot [antispam](https://kaworu.ch/blog/2014/03/25/dovecot-antispam-with-rspamd/) plugin nor do we use [imap_sieve](https://wiki.dovecot.org/HowTo/AntispamWithSieve).

So how does one tell rspamd that a mail is spam or ham? Other than of course running the commands in the terminal. We should at least document that.

**Comment by r-raymond** [comment]
I've never bothered training my spam filter. If you think its worth having those features, I can look into integrating them (of course PRs are also welcome :))

**Comment by phdoerfler** [comment]
Yes I believe it is worth to train your rspamd. Eventually https://github.com/r-raymond/nixos-mailserver/issues/61 will take care of the training with imap_sieve.
But as far as I understand it, this only works for new mail moved to the Junk mailbox. For the initial import you'd still have to resort to using the command line. And for this maybe a wiki page would be nice. You planned on putting the big readme.md into the wiki anyway, right?

**Comment by r-raymond** [comment]
You are correct, this is the plan. Feel free to create a wiki
page / snippet on how to do the command line training. I'll make sure
people can find it.
